src/driftpy/drift_client.py

```diff
@@ -158,7 +161,10 @@ def __init__(
             )
         else:
             self.tx_sender = (
-                StandardTxSender(self.connection, opts)
+                StandardTxSender(
+                    self.connection,
+                    opts,
+                    blockhash_commitment=tx_sender_blockhash_commitment if tx_sender_blockhash_commitment is not None else 'finalized')
```

Comment:
defaulting to 'finalized' for fetching blockhash because of too many 'blockhash not found' errors